src/main/resources/view/ModifiedAgenda.css

```diff
@@ -1,5 +1,5 @@
-.Agenda { 
-	-fx-font-family: "Segoe UI"; 	
+.Agenda {
+	-fx-font-family: "Segoe UI";
```

Comment:
What's up with this edit? Looks like the same.

Reply:
just whitespace for checkStyle